tests/rocprof-sys-videodecode-tests.cmake

```
@@ -0,0 +1,34 @@
+# -------------------------------------------------------------------------------------- #
+#
+# video decode tests
+#
+# -------------------------------------------------------------------------------------- #
+
+rocprofiler_systems_add_test(
+    SKIP_BASELINE SKIP_RUNTIME
+    NAME videodecode
+    TARGET videodecode
+    RUN_ARGS -i ${ROCmVersion_DIR}/share/rocdecode/video/ -t 2
+    REWRITE_ARGS
+        -e
+        -v
+        2
+        -R
+        run
+        --allow-overlapping
+        --print-available
+        functions
+        --print-overlapping
+        functions
+        --print-excluded
+        functions
+        --print-instrumented
+        functions
+        --print-instructions
+    LABELS "videodecode")
+
+rocprofiler_systems_add_validation_test(
+    NAME videodecode-perfetto-sampling
+    PERFETTO_FILE "perfetto-trace.proto"
+    LABELS "videodecode"
+    ARGS --key-names vaInitialize --key-counts 1)
```

```suggestion
    PERFETTO_METRIC "host"
    PERFETTO_FILE "perfetto-trace.proto"
    LABELS "videodecode"
    ARGS -l videodecode -c 1 -d 0)
```
Updated the validation parameters, but this will just be a basic "did it run" type of test. We'll have to modify the validate-perfetto-proto.py (or create a new one) to validate that the VCN counters are present.

---

You are right ... I missed including this change ...